watchtower-plugin/tests/test.py

```diff
@@ -48,7 +48,7 @@ def test_watchtower(node_factory, bitcoind, teosd):
     locator = change_endianness(dispute_txid[32:])
 
     # Make sure l2's normal penalty_tx doesn't reach the network
-    l2.daemon.rpcproxy.mock_rpc("sendrawtransaction", lambda: None)
+    l2.daemon.rpcproxy.mock_rpc("sendrawtransaction", lambda _: {"result": None ,"error": None,"id":"pytest"})
```

Comment:
Can we properly format this dict:
`{"result": None, "error": None, "id": "pytest"}`

And, if this RPC call is done multiple times, is it fine for all the answers to have the same "id" ("pytest")?

Reply:
Properly format as in this is missing a space?

And yeah, the id is just fine. In `bitcoin-cli` commands for `curl` they always use "curl"

Reply:
> Properly format as in this is missing a space?

Aha + comma position

Reply:
Sure, I guess it'll be worth adding a `black -l 120` check to the python files in the lint CI.